src/core/PWScore.cpp

```diff
@@ -731,7 +731,7 @@ struct ExportRecordWriter {
 
 private:
   PWSfile *m_pout;
-  PWScore *m_pcore;
+  /*PWScore *m_pcore; // variable not used */
```

Comment:
How about removing unused code?

Reply:
I had not been sure for unix or windows compilation, so stay with this as long no negative feedback is given.